Informer/Program.cs

```diff
@@ -7,8 +7,19 @@
 using Informer.Repository.Sqlite.Repositories;
 using Microsoft.EntityFrameworkCore;
 
+var MyAllowSpecificOrigins = "_myAllowSpecificOrigins";
+
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy(name: MyAllowSpecificOrigins,
+                      policy =>
+                      {
+                          policy.WithOrigins("http://localhost:5174");
```

Comment:
Nope! Please keep in mind that the port might change anytime the front-end developer needs to. We shouldn't have to change back-end's codes each time we want the front-end app to listen on another port. we should allow all ports on localhost or 127.0.0.1 in my opinion. But also, we should only allow this if we are running the app in development (or debug) environment and not in production